ngx-fudis/projects/ngx-fudis/src/lib/components/form/text-input/text-input.component.spec.ts

```diff
@@ -78,6 +80,21 @@ describe('TextInputComponent', () => {
     expect((component as any)._setComponentId).toHaveBeenCalledTimes(1);
   });
 
+  it('should destroy the component successfully', () => {
+    fixture.detectChanges();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    expect((component as any)._subscription.closed).toBeFalsy();
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    expect((component as any)._baseSubscription.closed).toBeFalsy();
+
+    fixture.destroy();
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    expect((component as any)._subscription.closed).toBeTruthy();
```

Comment:
This is also duplicate, previously written test can be removed from lines 133-142.

Reply:
Also I would suggest adding eslint-disable rule for the whole file instead of cluttering.